manifests/planning/ompl.xml

```diff
@@ -10,4 +10,5 @@
   <url>http://ompl.kavrakilab.org</url>
   <depend package="boost" />
   <depend package="flann" optional="1" />
+  <depend package="simulation/ode" />
```

Comment:
Unless I am mistaken, this adds a dependency of the rock package set to the rock.simulation package set.

I would suggest adding a simulation/ode osdep in the rock package set, and fixing the add_osdeps_override line in rock.simulation (which is currently completely bogus).